docs/source-fabric/index.rst

```diff
@@ -60,25 +60,39 @@ Fabric is the fast and lightweight way to scale PyTorch models without boilerpla
 Why Fabric?
 ***********
 
-Fabric differentiates itself from a fully-fledged trainer like Lightning :class:`pytorch_lightning.Trainer` in these key aspects:
+Fabric differentiates itself from a fully-fledged trainer like Lightning's `Trainer <https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html>`_ in these key aspects:
 
 **Fast to implement**
 There is no need to restructure your code: Just change a few lines in the PyTorch script and you'll be able to leverage Fabric features.
 
 **Maximum Flexibility**
 Write your own training and/or inference logic down to the individual optimizer calls.
-You aren't forced to conform to a standardized epoch-based training loop like the one in Lightning :class:`pytorch_lightning.Trainer`.
+You aren't forced to conform to a standardized epoch-based training loop like the one in Lightning `Trainer <https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html>`_.
 You can do flexible iteration based training, meta-learning, cross-validation and other types of optimization algorithms without digging into framework internals.
 This also makes it super easy to adopt Fabric in existing PyTorch projects to speed-up and scale your models without the compromise on large refactors.
 Just remember: With great power comes a great responsibility.
 
 **Maximum Control**
-The Lightning :class:`pytorch_lightning.Trainer` has many built-in features to make research simpler with less boilerplate, but debugging it requires some familiarity with the framework internals.
+The Lightning `Trainer <https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html>`_ has many built-in features to make research simpler with less boilerplate, but debugging it requires some familiarity with the framework internals.
 In Fabric, everything is opt-in. Think of it as a toolbox: You take out the tools (Fabric functions) you need and leave the other ones behind.
 This makes it easier to develop and debug your PyTorch code as you gradually add more features to it.
 Fabric provides important tools to remove undesired boilerplate code (distributed, hardware, checkpoints, logging, ...), but leaves the design and orchestration fully up to you.
 
 
+----
+
+************
+Installation
+************
+
+Fabric ships directly with Lightning. Install it with
+
+.. code-block:: bash
+
+    pip install lightning
+
+For alternative ways to install, read the `installation guide <https://pytorch-lightning.readthedocs.io/en/stable/starter/installation.html>`_.
```

Comment:
We can also have a separate installation page in the fabric docs, but for now, this is a quick fix just to get users started. 🛹